securedrop/i18n.py

```diff
@@ -211,11 +211,11 @@ def get_locale(config: SecureDropConfig) -> str:
     - config.DEFAULT_LOCALE
     - config.FALLBACK_LOCALE
     """
-    preferences = []
+    preferences: List[str] = []
     if session and session.get("locale"):
-        preferences.append(session.get("locale"))
+        preferences.append(str(session.get("locale")))
```

Comment:
I would suggest a fix like https://github.com/freedomofpress/securedrop/commit/f334468fe89f670d2a96e61ba2dc0eb93e7c1c67#diff-805c9c6b4ac8e1734ec16c741576801472ffc790bb11c6e58f3506cc9de76002L214-R218

Reply:
Thanks!  Amended in 7418177354540df81a9fdb900fc5e12d33e78473.